### PR TITLE
pktdump: add icmpv6 print

### PR DESCRIPTION
**Pull request by smlng**
print basic ICMPv6 header in packet dump

### CODE REVIEW

File: sys/net/gnrc/pktdump/gnrc_pktdump.c [hunk 1]
```diff
@@ -71,6 +72,7 @@ static void _dump_snip(gnrc_pktsnip_t *pkt)
 #ifdef MODULE_GNRC_ICMPV6
         case GNRC_NETTYPE_ICMPV6:
             printf("NETTYPE_ICMPV6 (%i)\n", pkt->type);
```
Comment (miri64) [inline sys/net/gnrc/pktdump/gnrc_pktdump.c]: The print of type here is now redundant

Comment (miri64) [inline sys/net/gnrc/pktdump/gnrc_pktdump.c]: Sorry, my bad. It's the nettype

File: sys/net/network_layer/icmpv6/icmpv6_hdr_print.c [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_pktdump
+ * @{
+ *
+ * @file
+ * @brief       Generic module to dump packages received via netapi to STDOUT
```
Comment (miri64) [inline sys/net/network_layer/icmpv6/icmpv6_hdr_print.c]: C-files don't need a `@brief` (it's not parsed by doxygen anyways) and the description is so generic I wouldn't even see it as internal doc.

**Comment by smlng** [comment]
addressed comments

**Comment by smlng** [comment]
@miri64 are you okay with this one, so I can squash?

File: sys/net/network_layer/icmpv6/icmpv6_hdr_print.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
```
Comment (miri64) [inline sys/net/network_layer/icmpv6/icmpv6_hdr_print.c]: You were not supposed to remove the `@author` tag... Please move the author back from the copyright line to that tag.

Comment (smlng) [inline sys/net/network_layer/icmpv6/icmpv6_hdr_print.c]: well I copy-pasted that from [ipv6_hdr_print.c](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/network_layer/ipv6/hdr/ipv6_hdr_print.c) to be consistent

Comment (miri64) [inline sys/net/network_layer/icmpv6/icmpv6_hdr_print.c]: The rest of the code has the `@author` flag. So ipv6_hdr_print.c is wrong ;-)

Comment (smlng) [inline sys/net/network_layer/icmpv6/icmpv6_hdr_print.c]: well then, I'll add the author tag here (and the other PR) and squash

**Comment by smlng** [comment]
@miri64 ping!

**Pull request merged**